.github/workflows/src/summarize-checks/labelling.js

```diff
@@ -158,6 +393,366 @@ export const breakingChangesCheckType = {
+  ciRpaasRPNotInPrivateRepoLabelShouldBePresent,
+  isNewApiVersion,
+  isDraft,
+) {
+  console.log("ENTER definition processARMReview")
+
+  const armReviewLabel = new Label("ARMReview", labelContext.present);
+  // By default this label should not be present. We may determine later in this function that it should be present after all.
+  armReviewLabel.shouldBePresent = false;
+
+  const newApiVersionLabel = new Label("new-api-version", labelContext.present);
+  // By default this label should not be present. We may determine later in this function that it should be present after all.
+  newApiVersionLabel.shouldBePresent = false;
+
+  const branch = targetBranch;
+  const isReleaseBranchVal = isReleaseBranch(branch)
+
+  // we used to also calculate if the branch name was from ShiftLeft, in which case we would or that
+  // with isReleaseBranchVal to see if it's in scope of ARM review. ShiftLeft is not supported anymore,
+  // so we only check if the branch is a release branch.
+  // const prTitle = await getPrTitle(owner, repo, prNumber)
+  // const isShiftLeftPRWithRPSaaSDevVal = isShiftLeftPRWithRPSaaSDev(prTitle, branch)
+  const isBranchInScopeOfSpecReview = isReleaseBranchVal// || isShiftLeftPRWithRPSaaSDevVal
+
+  // 'specReviewApplies' means that either ARM or data-plane review applies. Downstream logic
+  // determines which kind of review exactly we need.
+  let specReviewApplies = !isDraft && isBranchInScopeOfSpecReview
+  if (specReviewApplies) {
+    if (isNewApiVersion) {
+      // Note that in case of data-plane PRs, the addition of this label will result
+      // in API stewardship board review being required.
+      // See requiredLabelsRules.ts.
+      newApiVersionLabel.shouldBePresent = true;
+    }
+
+    armReviewLabel.shouldBePresent = resourceManagerLabelShouldBePresent
+    await processARMReviewWorkflowLabels(
+      labelContext,
+      armReviewLabel.shouldBePresent,
+      versioningReviewRequiredLabelShouldBePresent,
+      breakingChangeReviewRequiredLabelShouldBePresent,
+      ciNewRPNamespaceWithoutRpaaSLabelShouldBePresent,
+      rpaasExceptionLabelShouldBePresent,
+      ciRpaasRPNotInPrivateRepoLabelShouldBePresent)
+  }
+
+  newApiVersionLabel.applyStateChange(labelContext.toAdd, labelContext.toRemove)
+  armReviewLabel.applyStateChange(labelContext.toAdd, labelContext.toRemove)
+
+  console.log(`RETURN definition processARMReview. `
+    + `isReleaseBranch: ${isReleaseBranchVal}, `
+    + `isBranchInScopeOfArmReview: ${isBranchInScopeOfSpecReview}, `
+    + `isNewApiVersion: ${isNewApiVersion}, `
+    + `isDraft: ${isDraft}, `
+    + `newApiVersionLabel.shouldBePresent: ${newApiVersionLabel.shouldBePresent}, `
+    + `armReviewLabel.shouldBePresent: ${armReviewLabel.shouldBePresent}.`)
+
+  return { armReviewLabelShouldBePresent: armReviewLabel.shouldBePresent }
+}
+
+/**
+ * @param {string} branchName
+ * @returns {boolean}
+ */
+function isReleaseBranch(branchName) {
```

Comment:
If used by other code, should probably move into specModel.  Fine in here for now.